packages/modal/src/Modal.tsx

```diff
@@ -0,0 +1,281 @@
+import React, { Fragment, useCallback, useRef, SetStateAction } from 'react';
+import PropTypes from 'prop-types';
+import Portal from '@leafygreen-ui/portal';
+import Icon, { Size } from '@leafygreen-ui/icon';
+import { useEventListener } from '@leafygreen-ui/hooks';
+import { css, cx } from '@leafygreen-ui/emotion';
+
+export const ModalSize = {
+  XXSmall: 'xxsmall',
```

Comment:
We need to resolve this before merging, but I think Fred and I need to talk about having this many sizes included. Much harder to take something away than to add it, and I'm not sure we need more than, say, 3 sizes.

Reply:
Makes sense there's also been a lot of talk about this with the designers lately